src/lib/util/server_http_impl.hpp

```diff
@@ -0,0 +1,482 @@
+// license:MIT
```

Comment:
I'm not sure we want to accept core code under a different license.  The intention is to make src/lib/util and src/osd entirely BSD-licensed to facilitate reuse by other emulators.